### ISSUE TITLE
C# -> VB: Return default converts to "default"

### ISSUE THREAD

**Issue by HugoRoss** [icsharpcode/CodeConverter#428]
### Input code
```C#
 int? foo = default;
```

### Erroneous output
```vbnet
Dim foo As Integer? = "default"
```

### Expected output
```vbnet
Dim foo As Integer? = Nothing
```

### Details
* According to the current web converter at https://codeconverter.icsharpcode.net

**Comment by HugoRoss** [comment]
(or also just ``Dim foo As Integer?`` instead of ``Dim foo As Integer? = Nothing`` as there are no unassigned variables in VB.NET)

**Comment by Saibamen** [comment]
Fixed in release 8.0.0
